accumulo-store/src/test/java/gaffer/accumulostore/integration/AccumuloVisibilityIT.java

```diff
@@ -0,0 +1,430 @@
+package gaffer.accumulostore.integration;
```

Comment:
These tests are really helpful and I think they would be useful for any Store implementation. Could we move them into the gaffer-integration-tests module and set the class to extend AbstractStoreIT. We could then reuse them for all Stores.